test/generators/ModelGenerators.scala

```diff
@@ -16,9 +16,14 @@
 
 package generators
 
+import models.Checkbox.Option1
 import models._
-import org.scalacheck.Arbitrary.arbitrary
 import org.scalacheck.{Arbitrary, Gen}
 
 trait ModelGenerators {
+
+  implicit lazy val arbitraryCheckbox: Arbitrary[Checkbox] =
+    Arbitrary {
+      Gen.oneOf(Seq(Option1))
```

Comment:
Instead of hardcoding to Option1, this could be something like Gen.oneOf(Checkbox.values.toSeq)